backend/src/main/java/com/bakdata/conquery/util/search/TrieSearch.java

```diff
@@ -205,6 +206,14 @@ public void addItem(T item, List<String> keywords) {
 				.forEach(kw -> doPut(kw, item));
 	}
 
+	public void addItem(T item, String keyword) {
```

Comment:
könntest du hier nicht einfach auf ein List.of(keyword) delegaten?

Reply:
die methode kann ganz weg ;)